index.d.ts

```diff
@@ -1,25 +1,24 @@
 import {PackageJson} from 'type-fest';
 
-declare namespace meow {
-	type FlagType = 'string' | 'boolean' | 'number';
+export type FlagType = 'string' | 'boolean' | 'number';
 
-	/**
+/**
 	Callback function to determine if a flag is required during runtime.
 
 	@param flags - Contains the flags converted to camel-case excluding aliases.
 	@param input - Contains the non-flag arguments.
 
 	@returns True if the flag is required, otherwise false.
 	*/
```

Comment:
Incorrect indentation.

Applies everywhere. Please look at the file: https://github.com/LitoMore/meow/blob/esm/index.d.ts